### ISSUE TITLE
[a11y]: Long links links can overflow their containers

### ISSUE THREAD

**Issue by lal65** [psu-online-education/components#36]
### Summary

It is somewhat common for email addresses to overflow containers on small viewports. Consider `admissions@worldcampus.psu.edu` at 360px in width:

<img width="338" height="637" alt="Image" src="https://github.com/user-attachments/assets/ab3ee83d-8d71-49ec-9a61-9ad8787ffdf9" />

### Page URL

https://test.worldcampus.psu.edu/degrees-and-certificates/penn-state-online-applied-statistics-masters-degree#how-to-apply

### Expected behavior

The link should be completely legible and should not overflow its container.

### Actual behavior

The link overflows its container and can be partially obscured by backgrounds that it was never intended to appear on.

### Steps to reproduce

1. Visit https://test.worldcampus.psu.edu/degrees-and-certificates/penn-state-online-applied-statistics-masters-degree#how-to-apply on a mobile device
2. Note the admissions email may overflow

### Severity (will be revised and adjusted during triage as needed)

Medium

### Operating system and version

Android / iOS

### Browser and version

All

### Assistive technology and version

N/A

### Related WCAG criteria (optional)

Success criterion 1.4.3: Contrast (Minimum) (AA)

Success criterion 1.4.12: Text Spacing (AA)

### Additional context

An `overflow-wrap` property may be able to help with this.

```css
[href^="mailto:"] {
  overflow-wrap: break-word;
  word-break: break-all;
}
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct.
